config/locales/en/custom_fields.yml

```diff
@@ -49,6 +49,7 @@ en:
     hints:
       community:
         settings:
+          top_menu_customizations: "Enter links in Markdown format. Example: [Link Name](https://example.com). If the link name matches an existing one it will override the default URL. A URL of 'none' will remove a link completely. All other text and markup will be ignored."
```

Comment:
Let's add this to the front to be a bit more explicit. "Use this setting to edit the links in the top menu."

Reply:
There should also be a translation entry above under `labels` for the field label. Let's use `Top Menu Customizations`. While we use `nav` in the code, "Menu" will be more recognizable to users.